skfuzzy/defuzzify/defuzz.py

```diff
@@ -86,10 +122,91 @@ def dcentroid(x, mfx, x0):
 
     """
     x = x - x0
-    u = (x * mfx).sum(axis=0).astype(float) / np.fmax(mfx.sum(axis=0),
-                                                      np.finfo(float).eps)
+    u = centroid(x,mfx)
```

Comment:
Nicely spotted! Shame on me for not reusing code. Why not just make it a one-liner? 

`return x0 + centroid(x, mfx)`